display/pfobjects.py

```diff
@@ -147,7 +158,15 @@ def __init__(self, description):
             if i == 0:
                 tppoint = description.p4().Vect()
             self.graphline_thetaphi.SetPoint(i, math.pi/2.-tppoint.Theta(), tppoint.Phi())
-        if abs(self.desc.pdgid()) in [11,13]:
+        if abs(self.desc.pdgid()) in [11]:
```

Comment:
too much repeated code. use a if to define linewidth and linecolor according to type, and define 

``
def set_graph_style(graph, width, color)
``

apart from that the existing code should stay